dockerfiles/theia/Dockerfile

```diff
@@ -70,6 +73,13 @@ RUN che:theia production
 # change permissions
 RUN find production -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \; 2>log.txt
 
+# Clone and build che-theia plugins
```

Comment:
I would make it part of the theia-generator (the clone will be done in init step)

and then the compilation would occurs as another step of theia-generator like 'build-plugins' or something else

Then it will use same flag that @evidolob is adding for dev mode (master branch) vs tagged version, etc